src/app/zap-templates/zcl/data-model/draft/types/thermostat.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0"?>
-<!--
+	<?xml version="1.0"?>
```

Comment:
This restyle is kind of questionable - it does not seem to quite understand a top level xml declaration.

Still unsure if we want this in general (formatters are good ... but does this really help that much)?